translations/tds61.js

```diff
@@ -22,7 +22,7 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2014, 2018 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2014, 2019 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
This is an interesting copyright update.  I can only assume that it should be `2014, 2018, 2019` but I could be wrong.  Maybe the `2018` was an errant number...